server/webapp/WEB-INF/applicationContext-acegi-security.xml

```diff
@@ -251,7 +251,7 @@
                 /admin/restful/configuration/**=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
                 /admin/pipelines/**=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
                 /admin/pipeline_group/**=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
-                /admin/templates/**=ROLE_SUPERVISOR,ROLE_TEMPLATE_SUPERVISOR
+                /admin/templates/**=ROLE_USER
```

Comment:
Actually, no. Still need the before filters. But still. Makes sense to have this.